vmdb/app/models/orchestration_stack.rb

```diff
@@ -8,7 +8,7 @@ class OrchestrationStack < ActiveRecord::Base
 
   has_ancestry
 
-  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "EmsCloud"
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ExtManagementSystem"
```

Comment:
@blomquisg I believe @bwei told me to move this to orchestration_stack_openstack_infra.rb and overwrite it, at least I think it was this place. Or similar relation.